flow_reorganize_symlinks/setup_folder.cwl

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+requirements:
+  - class: InlineJavascriptRequirement
+
+inputs:
+  MAIN_PATH:
+    type: string
+  LINK_DIR: 
+    type: string
+  ASSEMBLY_NAME:
+    type: string
+    
+baseCommand: [mkdir]
+arguments:
+  #Genome Assembly
+  - position: 1
+    prefix: -p
+    valueFrom: $(inputs.MAIN_PATH)/$(inputs.LINK_DIR)/1.Genome Assembly/$(inputs.ASSEMBLY_NAME)/agp
+  - position: 2
+    valueFrom: $(inputs.MAIN_PATH)/$(inputs.LINK_DIR)/1.Genome Assembly/$(inputs.ASSEMBLY_NAME)/Contigs
+  - position: 3
+    valueFrom: $(inputs.MAIN_PATH)/$(inputs.LINK_DIR)/1.Genome Assembly/$(inputs.ASSEMBLY_NAME)/Scaffolds
+  #Official or Primary Gene Set
+  - position: 4
+    valueFrom: $(inputs.MAIN_PATH)/$(inputs.LINK_DIR)/2.Official or Primary Gene Set
```

Comment:
@HsiuKangHuang can you add the annotations directory name to the end of this (something like `valueFrom: $(inputs.MAIN_PATH)/$(inputs.LINK_DIR)/2.Official or Primary Gene Set/$(inputs.tree[2])`

Reply:
Okay! I can add this.